dependency-check-suppressions.xml

```diff
@@ -49,4 +49,17 @@
     		<vulnerabilityName>CWE-862: Missing Authorization</vulnerabilityName>
   	</suppress>
 
+	<!-- Temporary suppression before fix -->
+	<suppress>
+		<cve>CVE-2021-44832</cve>
+		<cve>CVE-2021-42550</cve>
+		<cve>CVE-2021-22060</cve>
+		<cve>CVE-2022-23181</cve>
+	</suppress>
+
+	<suppress>
+		<packageUrl regex="true">^pkg:maven/org\.springframework\.security/spring\-security\-core@.*$</packageUrl>
+		<vulnerabilityName>CWE-862: Missing Authorization</vulnerabilityName>
+	</suppress>
```

Comment:
Well, I wouldn't think new suppressions would be needed, as the master build was very recently green.